oxd-server/src/test/resources/testng.xml

```diff
@@ -5,6 +5,7 @@
     <parameter name="host" value="http://localhost"/>
     <parameter name="opHost" value="https://${test.server.name}"/>
     <parameter name="opDiscoveryPath" value=""/>
+    <parameter name="isLoadTest" value="false"/>
```

Comment:
create separate `testng-load.xml`. `isLoadTest` should not be here.

Reply:
Done